flow360/component/volume_mesh.py

```diff
@@ -955,6 +955,14 @@ def submit(self, description="", progress_callback=None, compress=True) -> Volum
                 renamed_file_on_remote, self.file_name, progress_callback=progress_callback
             )
 
+        if mesh_parser.is_ugrid() and os.path.isfile(mesh_parser.get_associated_mapbc_file()):
+            remote_mesh_parser = MeshNameParser(renamed_file_on_remote)
+            volume_mesh._webapi._upload_file(
+                remote_mesh_parser.get_associated_mapbc_file(),
+                mesh_parser.get_associated_mapbc_file(),
+                progress_callback=progress_callback,
+            )
```

Comment:
Can you add logging here to notify user when they submit a ugrid but we cannot find its mapbc file?

Reply:
added